### ISSUE TITLE
Not able to run Netopeer2 and sysrepo on CentOS 7

### ISSUE THREAD

**Issue by ChinmayaAgarwal** [CESNET/netopeer2#578]
I am trying to compile sysrepo (https://github.com/sysrepo/sysrepo/archive/v0.7.9.tar.gz) and netopeer2 (https://github.com/CESNET/Netopeer2/archive/v0.7-r2.tar.gz) along with their dependencies. The compilation shows no error but when i try to run sysrepo and netopeer2-server following error is thrown:-
[root@localhost ~]# sysrepod
sysrepod: error while loading shared libraries: libredblack.so.0: cannot open shared object file: No such file or directory
[root@localhost ~]# netopeer2-server
netopeer2-server: error while loading shared libraries: libssh.so.4: cannot open shared object file: No such file or directory
[root@localhost ~]# netopeer2-cli
netopeer2-cli: error while loading shared libraries: libssh.so.4: cannot open shared object file: No such file or directory

Same steps when i did on a ubuntu machine, it worked fine and everything runs.

**Comment by michalvasko** [comment]
Hi,
try running `ldconfig` or simply use it to check the status of the libraries (`ldconfig -p | grep libredblack`). In any case, this is not an issue with *netopeer2*.

Regards,
Michal

**Comment by ChinmayaAgarwal** [comment]
I did that but still i get the same error. Is this the issue because of PKG_CONFIG_PATH because when if try to compile protobuf-c, it throws an error that protobuf package not found . I corrected that by executing "export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH". Also, in centos dependency packages (.pc files) being downloaded on different paths where as in ubuntu it downloads at a single path i.e. /usr/local/lib/pkgconfig
